### PR TITLE
Fixed typos and formatting issues.

### PR DESCRIPTION
**Pull request by moxiegirl**
Carry and close #1852 

Signed-off-by: Rolfe Dlugy-Hegwer <rollfedh-contractor@docker.com>

### CODE REVIEW

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
         manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
-2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
+2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container that functions as the primary manager on `manager`.
```
Comment (thaJeztah) [inline docs/install-w-machine.md]: nit: "as the primary manager on `manager`" is a bit awkward (no show-stopper)

File: docs/install-w-machine.md [hunk 2]
```diff
@@ -172,21 +172,22 @@ Here, you connect to the cluster and review information about the Swarm manager
...
+        0b0628349187        hello-world         "/hello"                 20 minutes ago      Exited (0) 11 seconds ago                       agent1
         .
         .
         .
 
-    In this case, the Swarm ran 'hello-world' on the 'swarm1'.
+    In this case, the Swarm ran `hello-world` on `agent1`.
 
-    By default, Docker Swarm uses the "spread" strategy to choose which node runs a container. When you run multiple containers, the spread strategy assigns each container to the node with the fewest containers.
+    > By default, Docker Swarm uses the "spread" strategy to choose which node runs a container. When you run multiple containers, the spread strategy assigns each container to the node with the fewest containers.
```
Comment (thaJeztah) [inline docs/install-w-machine.md]: Should this be a "note"?

**Comment by vieux** [comment]
LGTM once @thaJeztah comments are fixed/answered

File: docs/install-w-machine.md (hunk 1 above)
Comment (moxiegirl) [inline docs/install-w-machine.md]: that's fine. we will stet this.

**Comment by thaJeztah** [comment]
LGTM (thanks!)

**Comment by moxiegirl** [comment]
Thanks folks. Docs break is unrelated.

**Pull request merged**